api/src/State/VersionProvider.php

```diff
@@ -10,11 +10,11 @@
 use App\Factory\DownloaderFactory;
 use Symfony\Contracts\Cache\TagAwareCacheInterface;
 
-readonly class VersionProvider implements ProviderInterface
+class VersionProvider implements ProviderInterface
 {
 
     public function __construct(
-        private(set) DownloaderFactory $downloaderFactory,
```

Comment:
This was not part of my request, do not make such changes please, please revert this line change